{
  "name": "merge-error-cause",
  "version": "3.5.0",
  "type": "module",
  "exports": {
    "types": "./build/src/main.d.ts",
    "default": "./build/src/main.js"
  },
  "main": "./build/src/main.js",
  "types": "./build/src/main.d.ts",
  "files": [
    "build/src/**/*.{js,json,d.ts}",
    "!build/src/**/*.test.js",
    "!build/src/{helpers,fixtures}"
  ],
  "scripts": {
    "test": "gulp test"
  },
  "description": "Merge an error with its inner cause",
  "keywords": [
    "nodejs",
    "javascript",
    "stacktrace",
    "library",
    "typescript",
    "browser",
    "errors",
    "monitoring",
    "error-monitoring",
    "message",
    "error-handler",
    "properties",
    "error-handling",
    "error",
    "code-quality",
    "cause",
    "error-reporting",
    "error-classes",
    "merge",
    "exceptions"
  ],
  "license": "MIT",
  "homepage": "https://www.github.com/ehmicky/merge-error-cause",
  "repository": "ehmicky/merge-error-cause",
  "bugs": {
    "url": "https://github.com/ehmicky/merge-error-cause/issues"
  },
  "author": "ehmicky <ehmicky@gmail.com> (https://github.com/ehmicky)",
  "directories": {
    "lib": "src",
    "test": "test"
  },
  "dependencies": {
    "normalize-exception": "^2.11.0",
    "set-error-class": "^1.5.0",
    "set-error-props": "^4.0.0",
    "wrap-error-message": "^1.0.0"
  },
  "devDependencies": {
    "@ehmicky/dev-tasks": "^2.0.15",
    "test-each": "^5.6.0"
  },
  "engines": {
    "node": ">=14.18.0"
  }
}
